### DISCUSSION TITLE
使用DBContext Remove后，怎么避免GUID主键被置为空

### DISCUSSION THREAD

**Discussion by zx1992147** [dotnetcore/FreeSql#1001] [Q&A]
我在使用DBSet.Remove后，会将Entity的ID置为空，这个可以避免么？
![image](https://user-images.githubusercontent.com/56952330/148480109-3b1cd256-88e8-468d-8641-317275482911.png)

**Comment by 2881099** [comment]
这个机制是为了下一次Add，暂时不行
